smoke.md

```diff
@@ -0,0 +1,303 @@
+# SMOKE 4.8.1 Porting Guide(CentOS 7.6)
+## Introduction
+The Sparse Matrix Operator Kernel Emissions (SMOKE) modeling system is primarily an emissions processing system designed to create gridded, speciated, hourly emissions for input into a variety of air quality models, such as CMAQ, REMSAD, CAMX, and UAM. SMOKE supports area, biogenic, mobile (both onroad and nonroad), and point source emissions processing for criteria, particulate, and toxic pollutants. For biogenic emissions modeling, SMOKE uses the Biogenic Emission Inventory System, version 2.5 (BEIS2) and version 3.09 and 3.14 (BEIS3). SMOKE is also integrated with the onroad emissions model MOBILE6 and MOVES.
+
+For more information about the SMOKE, visit the official SMOKE website.
+
+Programming language: Fortran 90
+
+Brief description: source emission tool for air modeling using sparse matrix
+
+Open-source protocol: LGPL-3.0
+
+##### Recommended Version
+The recommended version is SMOKE V4.8.1
+## Environment Requirements
+### Software Requirements
+| Item  | Version  |  Download Address |
+| ------------ | ------------ | ------------ |
+|  HDF5 |  1.10.1 | https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.1/src/hdf5-1.10.1.tar.gz  |
+|  PNETCDF |  1.8.0 |  https://parallel-netcdf.github.io/Release/parallel-netcdf-1.8.0.tar.gz |
+|  NETCDF-C | 4.4..1.1  |  https://github.com/Unidata/netcdf-c/archive/refs/tags/v4.4.1.1.tar.gz |
+| NETCDF-F  | 4.4.1  | https://github.com/Unidata/netcdf-fortran/archive/refs/tags/v4.4.1.tar.gz  |
+|  IOAPI | 3.2  | https://github.com/cjcoats/ioapi-3.2  |
+|  SMOKE | smoke_v481.Linux2_x86_64ifort.tar.gz smoke_v481.nctox.data.tar.gz smoke_install_v481.csh  | https://github.com/CEMPD/SMOKE/releases  |
+### OS Requirements
+| Item  | Version  | How to Obtain  |
+| ------------ | ------------ | ------------ |
+|  CentOS | 7.6  |  https://www.centos.org/download/ |
+| Kernel  | 4.14.0-115  |  Included in the OS image. |
+## Configuring the Compilation Environment
+### Installing dependencies
+
+
+    yum install time -y
+    yum install curl* -y
+    yum install csh -y
+    yum install zlib* -y
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "source /opt/rh/devtoolset-9/enable" >> /etc/profile
+### Installing Open MPI
```

Comment:
Add Open MPI to the table with dependencies above

Reply:
done